### PR TITLE
FORMS-9362: loading icon behind fields

### PR DESCRIPTION
**Pull request by AnurudraS**
## Description

As discussed with Rishi, the sibling div should have the selector as the ID of form and selecting the div with the same and adding the loading class on it later removing as well with the same steps.
However the test cases were already written in command.js file therefore, I have made the corrections in the same for the updated logic.
the new updates has been added to the readme file as well.

## Related Issue
FORMS-9362: Loading Icon Behind Fields

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](http://opensource.adobe.com/cla.html).
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [x] I have added tests to cover my changes and the overall coverage did not decrease.
- [ ] All unit tests pass on CircleCi.
- [x] I ran all tests locally and they pass.

### CODE REVIEW

**Comment by AnurudraS** [comment]
Test cases were already present in command.js therefore, I have updated the test case according to the latest code.

File: ui.tests/test-module/libs/support/commands.js
Comment (AnurudraS) [inline ui.tests/test-module/libs/support/commands.js:1]: Test cases were already present in command.js therefore, I have updated the test case according to the latest code.

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js [hunk 1]
```diff
@@ -63,14 +63,16 @@
         const startTime = new Date().getTime();
         let elements = document.querySelectorAll(FormContainerV2.selectors.self);
         for (let i = 0; i < elements.length; i++) {
-            elements[i].classList.add(FormContainerV2.loadingClass);
+            let loaderToAdd = document.querySelector("[data-cmp-adaptiveform-container-loader='"+ elements[i].id + "']");
+            loaderToAdd.classList.add(FormContainerV2.loadingClass);
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js:67]: Please add a null check here, 

if (loaderToAdd) {
     loaderToAdd.classList.add(FormContainerV2.loadingClass);
}

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js [hunk 2]
```diff
@@ -63,14 +63,16 @@
...
             console.debug("Form loading started", elements[i].id);
         }
         function onInit(e) {
             let formContainer =  e.detail;
             let formEl = formContainer.getFormElement();
             setTimeout(() => {
-                formEl.classList.remove(FormContainerV2.loadingClass);
+                let loaderToRemove = document.querySelector("[data-cmp-adaptiveform-container-loader='"+ formEl.id + "']");
```
Comment (rismehta) [inline ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js:74]: Also add a safe check here

**Review by rismehta — CHANGES_REQUESTED** [review]
check changes

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js (hunk 1 above)
Comment (AnurudraS) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js:68]: added!

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js (hunk 2 above)
Comment (AnurudraS) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js:76]: added!

**Review by rismehta — CHANGES_REQUESTED** [review]
Please write a test case for the fix

File: ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js [hunk 3]
```diff
@@ -63,14 +63,20 @@
         const startTime = new Date().getTime();
         let elements = document.querySelectorAll(FormContainerV2.selectors.self);
         for (let i = 0; i < elements.length; i++) {
-            elements[i].classList.add(FormContainerV2.loadingClass);
+            let loaderToAdd = document.querySelector("[data-cmp-adaptiveform-container-loader='"+ elements[i].id + "']");
+            if(loaderToAdd){
```
[the remark this answers is not in the document]
Comment (AnurudraS) [reply ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/container/v2/container/clientlibs/site/js/formcontainerview.js:67]: done.

**Pull request merged**